src/form/bf_mx.jl

```diff
@@ -379,6 +379,21 @@ function variable_mc_load_power(pm::AbstractUBFModels; nw=nw_id_default)
 end
 
 
+
+"""
+The variable creation for generators in branch flow model.
+Delta generators always need an auxilary power variable (X) similar to delta loads.
+Wye generators however, don't need any variables.
+"""
```

Comment:
Please correct the docstring format to include the function string at the top, i.e., 

```julia
"""
    variable_mc_generator_power(pm::AbstractUBFModels; nw::Int=nw_id_default, bounded::Bool=true, report::Bool=true)

The variable creation for generators in branch flow model. Delta generators always need an auxilary power variable (X) similar to delta loads. Wye generators however, don't need any variables.
"""
```